test/component/scheduler/perf/util.go

```diff
@@ -109,54 +110,61 @@ func makeNodes(c client.Interface, nodeCount int) {
 	}
 }
 
-// makePods will setup specified number of scheduled pods.
-// Currently it goes through scheduling path and it's very slow to setup large number of pods.
-// TODO: Setup pods evenly on all nodes and quickly/non-linearly.
-func makePods(c client.Interface, podCount int) {
-	glog.Infof("making %d pods", podCount)
-	basePod := &api.Pod{
+func makePodSpec() api.PodSpec {
+	return api.PodSpec{
+		Containers: []api.Container{{
+			Name:  "pause",
+			Image: "gcr.io/google_containers/pause:1.0",
+			Ports: []api.ContainerPort{{ContainerPort: 80}},
+			Resources: api.ResourceRequirements{
+				Limits: api.ResourceList{
+					api.ResourceCPU:    resource.MustParse("100m"),
+					api.ResourceMemory: resource.MustParse("500Mi"),
+				},
+				Requests: api.ResourceList{
+					api.ResourceCPU:    resource.MustParse("100m"),
+					api.ResourceMemory: resource.MustParse("500Mi"),
+				},
+			},
+		}},
+	}
+}
+
+// makePodsFromRC will create a ReplicationController object and
+// a given number of pods (imitating the controller).
+func makePodsFromRC(c client.Interface, name string, podCount int) {
```

Comment:
Not sure if this is a good name. It seems like the pods are not made from RC manager, but manually created for a RC. Personally I think makePodsForRC or even separate the creation of RC and its corresponding pods are better.